docs/aerodromes/classc/Edinburgh.md

```diff
@@ -4,7 +4,75 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name                | Callsign               | Frequency      | Login Identifier     |
+| ------------------- | ---------------------- | -------------- | -------------------- |
+| **Edinburgh ADC**   | **Edinburgh Tower**    | **118.300**    | **ED_TWR**           |
+| **Edinburgh SMC**   | **Edinburgh Ground**   | **134.100**    | **ED_GND**           |
+| Edinburgh ATIS      |                        | 126.250        | YPED_ATIS            |
+
+## Airspace
+<figure markdown>
+![ED ADC Airspace](img/edncircuits.png){ width="700" }
+</figure>
+
+- **ED ADC Airspace**: `SFC` to `A020` within the Edinburgh Control Zone (ED CTR). This airspace is primarily used for military circuits and initial and pitch approaches.
+
+## Runway Modes
+Single runway operations only.
+
+## Circuit Procedures
+The Edinburgh Circuit Area (ED CIRA) is active at all times when the Edinburgh CTR is active.
+
+Circuit operations occur within the CTR bounded by the Gawler River to the north and the powerline running from the Gawler River (1.5NM south of the Gawler Township) to the south eastern boundary of the ED CTR. Vertical Limits `SFC` to `A020`
+
+Aircraft can be instructed to extend outside of this airspace by ATC for traffic management.
+
+### Circuit Heights
+| Type    | Normal Altitude |
+| ------- | --------------- |
+| Non Jet | A015            |
+| Jet     | A020            |
+
+
+### Circuit Direction
+
+| Runway | Direction |
+| ------ | --------- |
+| 18     | Left      |
+| 36     | Left      |
+
+### Initial and Pitch Procedures 
+Aircraft will be pitching to the west in order not to impede on Class G boundary, the intail point is 2NM for RWY36 and 4NM for RWY18 along the extended centreline of taxiway E.
+
+## Helicopter Operations
+- **HLS Locations**: The HLS is located on TWY D immediately east of OLA 11. There are no SIDs associated with the HLS; aircraft requiring an instrument departure can expect to depart from the runway.
+<figure markdown>
+![ED HLS](img/ednhelicopterhls.png){ width="700" }
+</figure>
+
+## Coordination
+### Departures
+'Next' coordination is required from ED ADC to AD TCU for all aircraft.
+
+!!! phraseology
+    <span class="hotline">**ED ADC** -> **AD TCU**</span>: "Next, SGE"  
+    <span class="hotline">**AD TCU** -> **ED ADC**</span>: "SGE, unrestricted"  
+    <span class="hotline">**ED ADC** -> **AD TCU**</span>: "SGE"
+
+The Standard Assignable level from ED ADC to AD TCU is the lower of `A040` or the `RFL`.
```

Comment:
AD TMA page says this is RFL or A050, probably a typo on either one. @alphadelta332 you're probably in the know as to which one is correct

Reply:
@maxrumsey mentioned here https://github.com/vatpac-technology/sops/pull/413#discussion_r1898893997 that its A040. I'll change it on the AD TCU page too